presto-teradata-functions/src/test/java/com/facebook/presto/teradata/functions/TestTeradataDateFunctions.java

```diff
@@ -121,33 +135,34 @@ public void testEscapedText()
 
     private static SqlDate sqlDate(DateTime from)
     {
-        int days = (int) castToDate(SESSION.toConnectorSession(), from.getMillis());
+        int days = Ints.checkedCast(castToDate(SESSION.toConnectorSession(), from.getMillis()));
         return new SqlDate(days);
     }
 
-    private SqlTimestamp toTimestamp(DateTime dateTime)
+    private static SqlTimestamp toTimestamp(DateTime dateTime)
     {
         return new SqlTimestamp(dateTime.getMillis(), SESSION.getTimeZoneKey());
     }
 
+    @SuppressWarnings("SameParameterValue")
     private void assertTimestamp(String projection, int year, int month, int day, int hour, int minutes, int seconds)
     {
-        FUNCTION_ASSERTIONS.assertFunction(
+        functionAssertions.assertFunction(
```

Comment:
Thanks, all these can actually just be `assertFunction()` (doesn't even need `this`)